python/brayns/instance/client.py

```diff
@@ -33,9 +34,13 @@ def __init__(self, client: JsonRpcClient) -> None:
     def disconnect(self) -> None:
         self._client.disconnect()
 
-    def request(self, method: str, params: Any = None) -> Any:
```

Comment:
I think it would be good to keep a method that blocks until the task is completed. It could just call task() and then wait for the future to complete to avoid repeating code, but that way, the clients who does not need async does not have to wait on the futures.

Reply:
Nevermind, I saw that the request has been moved to instance now.

Reply:
It is already the case, but it is implemented in instance.py (base class of client). So I don't need to implement it for the mock.